uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup/leave-request-popup-comments-tab.html

```diff
@@ -1,19 +1,14 @@
 <crm-loading show="!$ctrl.loading.component">
   <div
-    class="text-center chr_leave-request-modal__comments_empty"
-    ng-show="commentsCtrl.getActiveComments().length === 0">
-    No comments available. Comments can be added below.
-  </div>
-  <div
     class="chr_leave-request-modal__comments"
     ng-hide="commentsCtrl.getActiveComments().length === 0">
     <div
       class="chr_leave-request-comment"
-      ng-repeat="comment in commentsCtrl.getActiveComments() | orderBy: commentsCtrl.orderComment : true">
+      ng-repeat="comment in commentsCtrl.getActiveComments() | orderBy: commentsCtrl.id : true">
       <div class="chr_leave-request-comment__meta">
         <span class="chr_leave-request-comment__author">{{commentsCtrl.getCommentorName(comment.contact_id)}}</span>
         <span>-</span>
-        <span class="chr_leave-request-comment__date">{{commentsCtrl.formatDateTime(comment.created_at)}}</span>
+        <span class="chr_leave-request-comment__date">{{comment.comment_id ? commentsCtrl.formatDateTime(comment.created_at) : "Just now"}}</span>
```

Comment:
it's better to add this to the controller so it can be tested. Something like this:

`{{ commentsCtrl.getCommentDate(comment); }}`

```js
vm.getCommentDate = function (comment) {
  comment.comment_id
  ? formatDateTime(comment.created_at)
  : 'Just now';
}
```